.github/workflows/php.yml

```diff
@@ -55,7 +58,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        presta-versions: ['1.7.8', 'latest', 'nightly']
+        presta-versions: ['nightly']
```

Comment:
Why remove those versions but not change them ? Wouldn't it make sense to test 8.0 & 8.1 ?

Reply:
Because, I remove support < PS 8.1.

`latest` is the latest release (8.0.1 at the moment).

Reply:
Because the `TranslatorAwareTrait` force the Symfony Contracts (and not anymore Symfony Component... which is a BC Break... in a minor version). (PR PrestaShop/PrestaShop#30496)

The `8.1` tag is not existing in Docker :whale:.